cmd/snap/cmd_run.go

```diff
@@ -111,7 +111,7 @@ and environment.
 			// TRANSLATORS: This should not start with a lowercase letter.
 			"gdb": i18n.G("Run the command with gdb (deprecated, use --gdbserver instead)"),
 			// TRANSLATORS: This should not start with a lowercase letter.
-			"gdbserver": i18n.G("Run the command with gdbserver"),
+			"gdbserver":              i18n.G("Run the command with gdbserver"),
```

Comment:
Did this land on purpose?

Reply:
not sure, I missed I was checking this in